chat/src/main/java/com/github/twitch4j/chat/ITwitchChat.java

```diff
@@ -79,6 +79,11 @@ default boolean sendMessage(String channel, String message, @Unofficial String n
     @Override
     void close();
 
+    /**
+     * @return the most recently measured round-trip latency for the socket(s) in milliseconds, or -1 if unknown
```

Comment:
Should a warning be added that this is the *socket* latency, which can be different from the latency of: sending a chat message/command & having it appear/processed by twitch

Reply:
The description describes what kind of latency this is, so this if fine imo.